docs/about/technical_steering_committee.md

```diff
@@ -1,55 +1,126 @@
 
 - Ensure that ongoing pull requests are moving forward at the right pace or closing them
 - Guide the community to use our various communication channels:
+    - [GitHub issues](https://github.com/kedro-org/kedro/issues) for feature requests and bug reports
+    - [GitHub discussions](https://github.com/kedro-org/kedro/discussions) to discuss the future of the Kedro project
+    - [Slack](https://slack.kedro.org) for questions and to support other users
 
-  - [GitHub issues](https://github.com/kedro-org/kedro/issues) for feature requests and bug reports
-  - [GitHub discussions](https://github.com/kedro-org/kedro/discussions) to discuss the future of the Kedro project
-  - [Slack](https://slack.kedro.org) for questions and to support other users
 
-## Requirements to become a maintainer
+## Process to become a maintainer
+Any current voting member may nominate a new candidate and will serve as their advocate, responsible for initiating and managing the voting process. The nominating member should open a pull request and formally call for a vote on the candidate, posting the announcement in the #kedro-tsc channel on the Kedro Slack organisation.
 
-Just contributing does not make you a maintainer; you need to demonstrate commitment to Kedro's long-term success by
-working with existing maintainers for a period of time.
+The voting period should remain open for at least two weeks, and the proposal will pass with a majority approval. Once voting concludes, the #kedro-tsc channel should be updated with the final results.
 
-We look for commitment markers who can do the following:
+The new maintainer should then be added to the `kedro-developers` team on the Kedro GitHub organisation,
+the `kedro-tsc` channel on the Kedro Slack organisation, the regular TSC meetings, and the `CITATION.cff` file.
 
-- Write high-quality code and collaborate with the team and community
-- Understand the project's code base and internals
-- Make pull requests from our backlog or roadmap; maintainers need to work towards a common goal
-- Learn how the team works, including processes for testing, quality standards and code review
-- Show evidence of already having started pull requests and code reviews under the guidance of maintainers; including asking
-  for help where needed
-- Show excitement about the future of Kedro
-- Build a collaborative relationship with the existing team
+### Inactivity and removal
+Maintainers who do not contribute code, documentation, or attend regular meetings for 6 months will be automatically removed from the [current TSC members](#current-kedro-tsc-members) list, `kedro-developers` team, and `CITATION.cff` file. They may be invited to join the [advisor](#advisors) role if they are interested in continuing to support and contribute to the project in a different capacity.
+
+
+## Responsibilities of an advisor
+
+- Providing ongoing, high-level guidance on direction, governance, and technical topics (rather than ad-hoc support)
```

Comment:
To play the devil's advocate: as a maintainer who has been contributing to the project for the past many years, why would I look for ongoing guidance on direction, governance, etc. from somebody who has not put the same time and effort into Kedro?  I would expect this guidance to come from the voting members of the TSC (and, conversely, voting members should be able to provide this sort of direction).

Reply:
By advisors, we mean people who are very knowledgeable in this space and work in MLOps or GenAI on a regular basis. These are practitioners whose day-to-day job is to put data pipelines into production and operate them in real environments.

The Kedro team (at McK) is largely made up of software engineers, and most of us don’t use Kedro or similar tools in production every day. Advisors bring that practitioner perspective and a different lens from those building the product day to day similar to how advisory boards work in startups.